tests/internal/test_database_monitoring.py

```diff
@@ -27,3 +27,64 @@ def test_injection_mode_configuration():
         excinfo.value.args[0] == "Invalid value for environment variable DD_TRACE_SQL_COMMENT_INJECTION_MODE: "
         "value must be one of ['disabled', 'full', 'service']"
     )
+
+
+@pytest.mark.subprocess(env=dict(DD_TRACE_SQL_COMMENT_INJECTION_MODE="disabled"))
+def test_get_dbm_comment_disabled_mode():
+    from ddtrace import tracer
+    from ddtrace.settings import database_monitoring
+
+    dbspan = tracer.trace("dbspan", service="orders-db")
+    sqlcomment = database_monitoring.get_dbm_comment(dbspan)
+    assert sqlcomment == ""
+
+
+@pytest.mark.subprocess(
+    env=dict(
+        DD_TRACE_SQL_COMMENT_INJECTION_MODE="service",
+        DD_SERVICE="orders-app",
+        DD_ENV="staging",
+        DD_VERSION="v7343437-d7ac743",
+    )
+)
+def test_get_dbm_comment_service_mode():
+    from ddtrace import tracer
+    from ddtrace.settings import database_monitoring
+
+    dbspan = tracer.trace("dbname", service="orders-db")
+
+    sqlcomment = database_monitoring.get_dbm_comment(dbspan)
+    # tags in sqlcomments are comma delimited
+    # assert 4 tags are present in the sqlcomment
+    assert len(sqlcomment.split(",")) == 4
+    # assert tags sqlcomment contains the correct value
+    assert "dddbs='orders-db'" in sqlcomment
```

Comment:
Nit: Would we want to import and use the keys we specified in database_monitoring.py instead of dddbs, dde, ddps, and ddpv?